src/main/java/domain/Ladder.java

```diff
@@ -0,0 +1,42 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import utils.NumberGenerator;
+
+public class Ladder {
+
+    private static final int NUMBER_OF_PEOPLE_TO_WIDTH_SCALE = 1;
+
+    private final List<Line> lines;
+    private final LadderHeight ladderHeight;
+
+    private Ladder(List<Line> lines, LadderHeight ladderHeight) {
+        this.lines = new ArrayList<>(lines);
+        this.ladderHeight = ladderHeight;
+    }
+
+    public static Ladder create(int numberOfPeople,
+                                LadderHeight ladderHeight,
+                                NumberGenerator numberGenerator) {
+        List<Line> lines = new ArrayList<>();
+        Ladder ladder = new Ladder(lines, ladderHeight);
+        ladder.addLines(numberOfPeople - NUMBER_OF_PEOPLE_TO_WIDTH_SCALE, numberGenerator);
+        return ladder;
+    }
+
+    private void addLines(int width, NumberGenerator numberGenerator) {
+        for (int i = 0; i < getLadderHeight(); i++) {
+            lines.add(Line.create(width, numberGenerator));
+        }
```

Comment:
i가 반복문 내에서 사용되고 있지 않네요. 

```suggestion
    var height = ladderHeight.getLadderHeight();
    while (height-- > 0) {
      lines.add(Line.create(width, numberGenerator));
    }
```
은 어떨까요?

Reply:
이걸 몰랐네요 !! 사용하지 않는 Index 정보를 제거할 수 있군요